rst2pdf/tests/input/sphinx-repeat-table-rows/index.rst

```diff
@@ -6,7 +6,7 @@ This table spans two pages. As the ``pdf_repeat_table_rows`` setting in
 2.
 
 .. table::
-   :widths:: 20 80
+   :widths: 20 80
```

Comment:
Wow, how'd I miss this in https://github.com/rst2pdf/rst2pdf/commit/4a795de61b1c3ab93a995c48e28a8ee0e4071e3c